src/sstruct_ls/ssamg_csolver.c

```diff
@@ -104,6 +109,7 @@ hypre_SSAMGCoarseSolverSetup( void *ssamg_vdata )
       HYPRE_BoomerAMGSetLogging(csolver, 1);
       //HYPRE_BoomerAMGSetAggNumLevels(csolver, 1);
       HYPRE_BoomerAMGSetup(csolver, par_Ac, par_b, par_x);
+      hypre_SetLogLevel(0);
```

Comment:
Why set the log level here?

Reply:
This is for resetting the log level so we don't get extra info about memory usage from the IJ calls. The goal was to get memory info only at the beginning and end of SSAMG and BoomerAMG setup (4 calls in total)